roles/patch_host_config/tasks/main.yml

```diff
@@ -50,6 +50,17 @@
       role: "{{ host_role }}"
   when: host_name is defined or host_role is defined
 
+- name: "Create patch body with hostname for {{ host_name }}"
+  set_fact:
+    patch_body:
+      hosts_names:
+        - "{{ host }}"
+  
+- name: "Add host role to patch body for {{ host_name }}"
+  set_fact:
+    patch_body: "{{ patch_body | combine({'hosts_roles': [host]}) }}"
```

Comment:
Not a blocker but the spacing is a bit strange here `combine({'hosts_roles': [host]}) }}"`.

Reply:
This better?
```
patch_body: "{{ patch_body | combine({ 'hosts_roles': [ host ] }) }}"
```

Reply:
+1